lsp-metals.el

```diff
@@ -594,6 +599,17 @@ WORKSPACE is the workspace we received notification from."
                             'local-map (lsp-metals--status-string-keymap workspace command?))
       workspace)))
 
+(lsp-defun lsp-metals--quick-pick (workspace (&MetalsQuickPickParams :items :place-holder?))
+  "The Metals quick pick request is sent from the server to the client to let the user provide a string value by picking one out of a number of given options."
+  (let* ((choices (seq-map (lambda (item)
+                             (-let* (((&MetalsQuickPickItem :id :label :description?) item))
+                               (cons (if description? (concat label " " (propertize description? 'face 'font-lock-comment-face)) label) id)))
```

Comment:
I'd suggest to use `affixation-function` from emacs 28 here. Current implementation matches user input by not only item but by description too.